flask_monitoringdashboard/templates/fmd_dashboard/overview.html

```diff
@@ -65,3 +65,8 @@
 
 {% endblock %}
 {% endblock %}
+{% block script %}
+  <script type="text/javascript">
+    $('#dataTable').DataTable().order([[3, "desc"]]).draw();
```

Comment:
You are able to do the sorting in the frontend because the frontend actually has all the data from the backend, right?

Reply:
that's right